pypit/arutils.py

```diff
@@ -1004,7 +1006,10 @@ def robust_polyfit(xarray, yarray, order, weights=None, maxone=True, sigma=3.0,
         yrng = func_val(ct, xarray, function, minv=minv, maxv=maxv)
         sigmed = 1.4826*np.median(np.abs(yfit-yrng[w]))
         if xarray.size-np.sum(mask) <= order+2:
-            msgs.warn("More parameters than data points - fit might be undesirable")
+            try:
```

Comment:
I'm not sure why this is in a try/except clause. Do you recall what the AttributeError was, and is it possible to fix it?